app/code/core/Mage/Core/Block/Abstract.php

```diff
@@ -1157,6 +1157,51 @@ public function formatTime($time = null, $format = Mage_Core_Model_Locale::FORMA
         return $helper->formatTime($time, $format, $showDate);
     }
 
+    public function getIconSvg(string $name, string $variant = 'outline'): string
+    {
+        $name = strtolower($name);
+        $variant = strtolower($variant);
+
+        $packageName = 'mahocommerce/icons';
+        $iconName = basename($name);
+        $allowedVariants = ['outline', 'filled'];
+
+        if (!in_array($variant, $allowedVariants)) {
+            $variant = 'outline';
+        }
+
+        $installPath = \Composer\InstalledVersions::getInstallPath($packageName);
```

Comment:
This can both return `null` and `throw new \OutOfBoundsException('Package "' . $packageName . '" is not installed')`.

Probably return empty string in both scenarios?

Reply:
right! done :-)